andela/ah-scorpion
# #159726516 Enable liking/disliking comments

**What does this PR do?**

Enable users to like or dislike a comment.

**Description of tasks completed**

- Wrote tests for the feature.
- Added the like and dislike fields in the database.
- Added the like and dislike endpoints with the appropriate views.
- Added the like and dislike count in the CommentSerializer.

**How can this PR be manually tested?**

- clone this branch.
- go to the url of a comment and append `/like/` e.g `api/v1/articles/<slug>/comments/<pk>/like/` and make a PUT request to like it.
- Repeating the above step will unlike the comment.
- go to the url of a comment and append `/dislike/` e.g `api/v1/articles/<slug>/comments/<pk>/dislike/` and make a PUT request to dislike it.
- Repeating the above step will un-dislike the article.
- run tests using `python manage.py test`. All tests should pass.

**What are the relevant pivotal tracker stories?**
[#159726516](https://www.pivotaltracker.com/story/show/159726516)
